workspaces/acs/plugins/acs/src/common/QueryACSData.ts

```diff
@@ -25,11 +25,11 @@ export const QueryACSData = (deploymentName: string) => {
   const [result, setResult] = useState([]);
   const [loaded, setLoaded] = useState<boolean>(false);
```

Comment:
nit: In our app, we usually use the term `isLoading` instead of `loaded`. Also, not something we need to refactor now but just worth noting.